src/arcade/src/Microsoft.DotNet.Arcade.Sdk/tools/PrebuiltUsage.targets

```diff
@@ -1,31 +1,32 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
-<Project DefaultTargets="WritePrebuiltData">
-
-  <Import Project="..\BuildStep.props" />
+<Project>
 
   <PropertyGroup>
     <MicrosoftDotNetSourceBuildTasksBuildDir>$(NuGetPackageRoot)microsoft.dotnet.sourcebuild.tasks\$(MicrosoftDotNetSourceBuildTasksVersion)\build\</MicrosoftDotNetSourceBuildTasksBuildDir>
   </PropertyGroup>
-
   <Import Project="$(MicrosoftDotNetSourceBuildTasksBuildDir)Microsoft.DotNet.SourceBuild.Tasks.props" />
 
-  <Target Name="WritePrebuiltData" Condition="'$(DotNetBuildInnerRepo)' != 'true'">
-    <ItemGroup>
-      <AllRestoredPackageFiles Include="$(CurrentRepoSourceBuildPackageCache)**/*.nupkg" />
-      <SourceBuiltPackageFiles Include="$(AdditionalSourceBuiltNupkgCacheDir)**/*.nupkg" Condition=" '$(AdditionalSourceBuiltNupkgCacheDir)' != '' " />
-      <SourceBuiltPackageFiles Include="$(ReferencePackageNupkgCacheDir)**/*.nupkg" Condition=" '$(ReferencePackageNupkgCacheDir)' != '' " />
-      <SourceBuiltPackageFiles Include="$(PreviouslySourceBuiltNupkgCacheDir)**/*.nupkg" Condition=" '$(PreviouslySourceBuiltNupkgCacheDir)' != '' " />
-
-      <!-- Add some other potential top-level project directories for a more specific report. -->
-      <ProjectDirectories Include="$(CurrentRepoSourceBuildSourceDir)"  Condition="'$(UseInnerClone)' == 'true'" />
-      <!-- Finally, scan entire source-build, in case project.assets.json ends up in an unexpected place. -->
-      <ProjectDirectories Include="$(SourceBuildOutputDir)" />
-    </ItemGroup>
+  <!-- Repo extensibility point. -->
+  <Import Project="$(RepositoryEngineeringDir)PrebuiltUsage.props" Condition="Exists('$(RepositoryEngineeringDir)PrebuiltUsage.props')" />
```

Comment:
Is any repo using this extensibility point?

Reply:
This is WIP. Not yet sure if we will need it.